coalib/bears/meta.py

```diff
@@ -0,0 +1,25 @@
+from collections import defaultdict
+
+
+class bearclass(type):
+    """
+    Metaclass for :class:`coalib.bears.Bear.Bear` and therefore all bear
+    classes.
+
+    Pushing bears into the future... ;)
+    """
+
+    # by default a bear class has no aspects
```

Comment:
I don't think this is a good idea. Aspects are supposed to be things that the bear can analyze and fix. Root is just the parent of all aspects. We don't expect the user to ever mention `Root` as the aspect to be analyzed. It makes no sense to put `Root` in `aspects.detect` as it is not some physical aspect of the code which the bear analyzes and is hence counter-productive. 
When bears are called eexplcitly, we would have to have a different bear collection methodology as when called via aspects so adding `Root` won't help much anyway.